### DISCUSSION TITLE
2023/10/15/java%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0%20%20%E6%9D%A5%E6%BA%90%E4%BA%8E%E5%BB%96%E9%9B%AA%E5%B3%B0java%E6%95%99%E7%A8%8B/

### DISCUSSION THREAD

**Discussion by giscus** [bigbigbigwaterbucket/bigbigbigwaterbucket.github.io#4] [Announcements]
# 2023/10/15/java%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0%20%20%E6%9D%A5%E6%BA%90%E4%BA%8E%E5%BB%96%E9%9B%AA%E5%B3%B0java%E6%95%99%E7%A8%8B/

科技不是一题一解的死板学识，而是令我们超凡脱俗的产物。

https://bigbigbigwaterbucket.github.io/2023/10/15/java%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0%20%20%E6%9D%A5%E6%BA%90%E4%BA%8E%E5%BB%96%E9%9B%AA%E5%B3%B0java%E6%95%99%E7%A8%8B/

**Comment by LuoYuXuanRyan** [comment]
首页不分页？💢
